DEVELOP.adoc

```diff
@@ -7,7 +7,7 @@ Some notes for developers and other people willing to help development.
 - Install:
 * tox
 * libacl-devel (for sys/acl.h)
-* rdiff (with support for -H parameter)
+* rdiff
```

Comment:
We need to see but there might be a merge conflict as the diff is based on your other MR and not on master.

Reply:
Forget it, your other change was already merged in master.